bin/rename_fasta.py

```diff
@@ -60,7 +60,7 @@ def main(
             cluster_file
         ), "Provided cluster information file does not exist"
     for file in files:
-        if file.endswith(("fa", "fasta", "fna")) and not file.startswith(prefix):
+        if file.endswith(("fa", "fasta", "fna")):
```

Comment:
Initially, it was to not rename genomes in the folder if they are already named MGYG*. This was written for the very first gut catalogue update when many genomes were MGYG's to begin with. But we don't use rename the same way anymore so we don't need that. Also, if any MGYG's are passed to the pipeline rather than their INSDC names, the pipeline will fail.

Reply:
Perfect, thanks for removing that then :)